CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 - Paths too long when unpacking (#2156)
+- Improved stability of saving app meta data file - caching files every 100MB
+- Saving executable name and arguments - only when changed
```

Comment:
I'd say it should not go under the `Fixed` category but `Changed`. Also, the changelog entry should reflect the clearly visible change - in this case speeding up the installation/patching process due to less frequent meta data saving.

Reply:
Done